src/index.ts

```diff
@@ -262,7 +261,7 @@ export async function decrease(incremental) {
     (incremental === "pitch" ||
       incremental === "rate" ||
       incremental === "volume") &&
-    oc(R2Navigator.rights).enableTTS(false)
+    (R2Navigator.rights?.enableTTS ?? false)
```

Comment:
do we need `?? false` o should that work without it , just this `R2Navigator.rights?.enableTTS`